src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -426,23 +474,23 @@ Check out our [curated Web Bluetooth Demos](https://github.com/WebBluetoothCG/de
 
 ## Dev Tips
 
-A Bluetooth Console is available in Chrome OS developer shell. Press [ Ctrl ] [
-Alt ] [ T ] to open a browser tab terminal and use the `bt_console` command to
-start poking around your bluetooth settings. The `help` command will give you a
-list of all available commands.
+A "Bluetooth Internals" page is available in the browser at
```

Comment:
Change 'in the browser' to 'in Chrome' (because we're talking specifically about Chrome).